Assets/MixedRealityToolkit/_Core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -327,6 +353,8 @@ public bool BoolData
                 }
 
                 Changed = boolData != value;
+                // use the internal reading for changed so we don't reset it.
+                Updated = changed || value;
```

Comment:
Firing `Pressed` events based on this logic might lead to confusing behavior.

For example, if I release the trigger, `changed` will be true but `value` will be false, and the system will fire an `InputUp` followed by an `InputPressed` (since `Updated` will be true).

It seems like we should only be firing `Pressed` events whenever the value isn't the default, instead of also whenever it changes.

Reply:
Makes sense. What changes do you think would need to happen to address this?

Reply:
I think just removing `changed ||` from each line fixes it? So then `Updated` (or whatever it gets called) will be true when the input is active / value isn't the default, and false otherwise. Then, `Changed` tells us when the new value != the old value, and `Updated` tells us when the action is "pressed" for the purposes of the Pressed events.

Reply:
Could you open a PR on my branch? I'm not sure I fully understand the changes you're requesting.

Thanks :)